Habitica/res/values/strings.xml

```diff
@@ -368,7 +369,7 @@ To start, which parts of your life do you want to improve?</string>
     <string name="share_hatched" formatted="false">I just hatched a %1$s %2$s pet in Habitica by completing my real-life tasks!</string>
     <string name="share_raised" formatted="false">I just gained a %1$s %2$s mount in Habitica by completing my real-life tasks!</string>
     <string name="open_in_store">Open in play store</string>
-    <string name="change_class_confirmation">Are you sure you want to change your class? This will cost 3 gems.</string>
+    <string name="change_class_confirmation">Are you sure you want to change your class? This will cost 3 gems</string>
```

Comment:
I think the full stop is best left in. I realise it wasn't there in the suggested text in the issue but we do want it here since it's a full sentence.

Can you please also include the warning from the issue about not being able to buy gear from the previous class?